code/modules/antagonists/cult/runes.dm

```diff
@@ -780,11 +777,11 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 /obj/effect/rune/blood_boil/proc/do_area_burn(turf/T, multiplier)
 	set_light(6, 1, color)
-	for(var/mob/living/L in viewers(T))
-		if(!IS_CULTIST(L) && L.blood_volume)
-			if(L.anti_magic_check(chargecost = 0))
+	for(var/mob/living/target in viewers(T))
+		if(!IS_CULTIST(target) && target.blood_volume)
+			if(target.can_block_magic(charge_cost = 0))
```

Comment:
This one doesn't check for `MAGIC_RESISTANCE_HOLY` but the first dies?

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/64124/commits/8b10a3c65e4b7d5217c9a344a97828348a9bf9ce good catch!